files/claude/skills/preflight/SKILL.md

```
@@ -0,0 +1,112 @@
+- [ ] Form inputs have associated labels
+
+### Code Style
+
+- [ ] No semicolons
+- [ ] Files end with single newline
+- [ ] Empty lines have no whitespace
+- [ ] No trailing whitespace
+
+## Output Format
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)